BreanaMarie/tic-tac-toe/ticTacToe.js

```diff
@@ -0,0 +1,458 @@
+console.log ("connected");
+// wait for the DOM to finish loading
+$(document).ready(function(){
+  // all code to manipulate the DOM
+  // goes inside this function
+
+
+
+var round='';
+var i=0;
+
+//turn counter x = even o = odd 
+//declare draw
+function turn(blah){
+	if ( i < 9){
+		//changeSpace();
+		if (i%2 ===0){
+			console.log("WDI 23's turn");
+			round ='WDI 23';
+		}
+		else if (i%2 !==0){
+			console.log("Space Jammer's turn");
+			round ='Space Jammer';
+		}
+	}
+	else if (i==10){
+		alert("Game is a Draw! There are no more space to play.");
+		resetSpaces();
+	}
+}
+
+//state Who's turn it is
+
+$("div.box").on('click', function() {
+	var text = round;
+	var who = "It is " + text + "s Turn";
+    $("p").html(who);
+});
+
+//$("#turnEdit").html("It is " + round +"'s Turn");
+
+//check if box is free or has been played
+function changeSpace(selectedSpace){
+	turn();
+	$('div.box').children().on('click', function(){
+		console.log($(this).text()+ " has been clicked");
+		selectedSpace = $(this);
+		console.log(selectedSpace);
+
+		//if freeSpace
+		if (selectedSpace.hasClass('freeSpace')){
+			console.log("the space is free");
+
+			//check turn
+			if (round ==='WDI 23'){
+				console.log("WDI 23 just went");
+				//change space to played
+			    if(selectedSpace.hasClass('freeSpace')) { 
+			        selectedSpace.removeClass('freeSpace');
+			        selectedSpace.addClass('playedSpaceX');
+			        //check if play won
+			        winSpaces();
+			        console.log (i);
+			        i ++;
+			        turn();
+		    	}	
+			}
+		    else if (round === 'Space Jammer'){
+		    	console.log("Space Jammer just went");
+		    	if(selectedSpace.hasClass('freeSpace')) { 
+			        selectedSpace.removeClass('freeSpace');
+			        selectedSpace.addClass('playedSpaceO');
+			        
+			        winSpaces();
+			        console.log(i);
+			        i ++;
+			        turn();
+		    	} 
```

Comment:
Between here and line 51, it seems like the code is almost identical.  When this happens you can often find a way to DRY out your code.  (Don't repeat yourself)  One way to do this might be to create a function that accepts arguments related to what the differences are.